app/components/reactions/reactions.js

```diff
@@ -69,45 +74,68 @@ export default class Reactions extends PureComponent {
 
         const addMoreReactions = (
             <TouchableOpacity
+                key='addReaction'
                 onPress={this.props.onAddReaction}
                 style={[styles.reaction]}
             >
-                <Text style={styles.more}>{'+'}</Text>
+                <Image
+                    source={addReaction}
```

Comment:
On first read, I thought its an action. Can we change it to `addReactionIcon`?

Reply:
Sure thing, I’ll do it as soon as I can.